### PR TITLE
Bull board

### PR DESCRIPTION
**Pull request by bearfact**

### CODE REVIEW

File: jobs/worker.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
-const Arena = require('bull-arena');
 const express = require('express');
+const { router, setQueues, BullAdapter } = require('bull-board')
```
Comment (aprakash-gc) [inline jobs/worker.js:2]: semicolon, but whatever

File: jobs/worker.js [hunk 2]
```diff
@@ -45,7 +28,8 @@ const register = () => {
 	app.set('port', parseInt(process.env.ARENA_PORT) || 4000);
 
 	app.use(arenaAuth);
```
Comment (aprakash-gc) [inline jobs/worker.js:30]: need `arenaAuth` still?

**Review by aprakash-gc — CHANGES_REQUESTED** [review]

File: jobs/worker.js (hunk 2 above)
Comment (bearfact) [reply jobs/worker.js:30]: yes, although I could rename it

Comment (bearfact) [reply jobs/worker.js:30]: I'd have to update all the env variables as well.  Gonna leave that for another pass.

**Pull request merged**